spec/src/main/asciidoc/license-efsl.adoc

```diff
@@ -0,0 +1,73 @@
+[subs="normal"]
+....
+Specification: {doctitle}
+Version: {revnumber}
+Status: {revremark}
+Release: {revdate}
+....
+
+== Copyright
+
+Copyright (c) 2016, 2020 Eclipse Foundation.
```

Comment:
You should use inceptionYear and currentYear here. See [here](https://github.com/eclipse/microprofile-fault-tolerance/blob/master/spec/src/main/asciidoc/license-efsl.adoc). It was further improved after Health was released.

Reply:
You need to copy the inceptionYear and currentYear declaration in pom.xml

Reply:
thanks, added the suggestion.